legend-pure-core/legend-pure-m3-core/src/test/resources/testMilestoningRepository/testMilestoning.pure

```diff
@@ -0,0 +1,36 @@
+// Copyright 2021 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+ Class <<temporal.businesstemporal>> meta::test::milestoning::A
+ {
+   a : String[1];
+ }
+
+ Class <<temporal.processingtemporal>> meta::test::milestoning::B
+ {
+   b : String[1];
+ }
+
+Class <<temporal.businesstemporal>> meta::test::milestoning::Account
+{
+}
+
+Class <<temporal.businesstemporal>> meta::test::milestoning::FirmAccount extends meta::test::milestoning::Account
+{
+}
+
```

Comment:
We should also have a bitemporal class and a non-milestoned class, and connections between each type. Also, properties from associations.